docs/src/reference/index.md

```diff
@@ -78,11 +83,6 @@ Check the index on the left for a more detailed description of any symbol.
 | [`tp.equal()`][temporian.equal] [`tp.not_equal()`][temporian.not_equal] [`tp.greater()`][temporian.greater] [`tp.greater_equal()`][temporian.greater_equal] [`tp.less()`][temporian.less] [`tp.less_equal()`][temporian.less_equal]               | Compute a relational binary operator between two [`EventSets`][temporian.EventSet].                                         |
```

Comment:
`tp.equal()` no longer exists right? I'd move all of the binary ones to the EventSet if we're moving `.equal()`. @achoum is there a reason we haven't already done this?

Reply:
I think the main reason is that we already have the magic methods for those. `evset.equal()` is needed because we can't overload the `==` operator.
IMO we should remove all these  from `__init__`, without adding them to `event_set_ops`. I don't see any use case where using these methods is a much better option, and we should try to not have two ways to do things.

(this was the last point I wanted to discuss today)

Reply:
LGTM from my side to remove them 👍🏼